portworx-Cassandra/1.8/README.md

```diff
@@ -0,0 +1,73 @@
+
+The following instruction will install Cassandra service on DC/OS cluster backed by PX volumes for persistent storage.
+
+
+# Prerequisites
+
+- A running DC/OS v1.8 cluster with at least 3 private agents with Portworx running on all three
+- Portworx works best when installed on all nodes in a DC/OS cluster.  If Portworx is to be installed on a subset of the cluster, then:
+  * the agent-nodes must include attributes indicating the participate in the Portworx cluster.
+  * services that depend on Portworx volumes must specify "constraints" to ensure they are launched on nodes that can access Portworx volumes.
+- A node in the cluster with a working DC/OS CLI.
+
+Please review the main [Portworx on DCOS](https://docs.portworx.com/scheduler/mesosphere-dcos/) documentation.
+
+# Install Cassandra
+## Adding repository to DC/OS cluster
+Login to a node which has the DC/OS CLI installed and is authenticated to the DC/OS cluster
+Run the following command to add the repository to the DC/OS cluster
+```
+ $ dcos package repo add --index=0 cassandra https://px-dcos.s3.amazonaws.com/v2/cassandra/cassandra.json
+```
+Now Cassandra-PX package should be available under Universe->Packages
+![Cassandra Package List](img/Cassandra-install-01.png)
+## Default Install
+If you want to use the defaults, you can now run the dcos command to install the service
+```
+ $ dcos package install --yes cassandra-px
```

Comment:
Package name will be portworx-cassandra. Same for the other services

Reply:
The screens shot still points to the package name as "cassandra-px" . What do you want me to do there?